### ISSUE TITLE
Uncaught TypeError: this.editor.displayBuffer.onDidCreateMarker is not a function

### ISSUE THREAD

**Issue by abe33** [atom-minimap/minimap-bookmarks#3]
```
TypeError: this.editor.displayBuffer.onDidCreateMarker is not a function
  at new MinimapBookmarksBinding (/Users/cedric/.atom/packages/minimap-bookmarks/lib/minimap-bookmarks-binding.coffee:11:46)
  at /Users/cedric/.atom/packages/minimap-bookmarks/lib/minimap-bookmarks.coffee:29:21
  at /Users/cedric/.atom/packages/minimap/lib/main.js:452:11
  at Map.forEach (native)
  at Main.observeMinimaps (/Users/cedric/.atom/packages/minimap/lib/main.js:451:30)
  at Object.module.exports.activatePlugin (/Users/cedric/.atom/packages/minimap-bookmarks/lib/minimap-bookmarks.coffee:27:38)
  at Main.activatePlugin (/Users/cedric/.atom/packages/minimap/lib/mixins/plugin-management.js:236:14)
  at Main.updatesPluginActivationState (/Users/cedric/.atom/packages/minimap/lib/mixins/plugin-management.js:219:16)
  at /Users/cedric/.atom/packages/minimap/lib/mixins/plugin-management.js:293:15
  at Config.module.exports.Config.observeKeyPath (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:612:7)
  at Config.module.exports.Config.observe (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:141:21)
  at Main.registerPluginControls (/Users/cedric/.atom/packages/minimap/lib/mixins/plugin-management.js:292:55)
  at Main.registerPlugin (/Users/cedric/.atom/packages/minimap/lib/mixins/plugin-management.js:116:14)
  at Object.module.exports.consumeMinimapServiceV1 (/Users/cedric/.atom/packages/minimap-bookmarks/lib/minimap-bookmarks.coffee:15:14)
  at Provider.module.exports.Provider.provide (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/provider.js:30:52)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/service-hub.js:55:26
  at doNTCallback0 (node.js:443:9)
  at process._tickCallback (node.js:372:13)
```

**Comment by fundon** [comment]
Updated to https://atom.io/docs/api/v1.9.9/MarkerLayer and https://atom.io/docs/api/v1.9.9/MarkerLayer.

**Comment by fundon** [comment]
Landed in v0.4.0.
